doc/readme/EXAMPLES.md

```diff
@@ -35,7 +35,7 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example: simple create another `*.sg` file.
+It is extremely simple to add a new example. Just create another `*.sg` file.
 We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
```

Comment:
Maybe you can write here the steps required to porting the examples? It is described in the issue, but would be easier to have it all here imo

Reply:
@gf712 Got it! I will add some steps as mentioned in the issue and will create a new PR. Thank you for your suggestions!